apps/src/music/views/MusicLabView.tsx

```diff
@@ -119,6 +121,21 @@ const MusicLabView: React.FunctionComponent<MusicLabViewProps> = ({
     }
   }, [blocklyWorkspace, isStartMode, levelData]);
 
+  // Use the Lab2 generic prompt for Blockly prompt dialogs.
+  const genericPrompt = useCallback(
+    (p1: string, p2: string, p3: (p1: string | null) => void) => {
+      dialogControl.showDialog({
+        type: DialogType.GenericPrompt,
+        title: '',
+        message: p1,
+        value: p2,
+        handleConfirm: p3,
+      });
+    },
+    [dialogControl]
+  );
+  Blockly.dialog.setPrompt(genericPrompt);
+
```

Comment:
Do the `p1`, `p2`, `p3` args have any significances to blockly?

Otherwise I'd recommend renaming them as just `message`, `value`, `handleConfirm` (or maybe `handleConfirmCallback`? Since the actual arg is the callback itself?) for clarity's sake.

Reply:
They came from Blockly, but there's no reason we need to keep them. Updated!

https://github.com/google/blockly/blob/5fd337bb4d12b68766cae8094893ed0131285666/core/dialog.ts#L110-L114